### PR TITLE
BaSP-TG-112: Update Layout routes

### PR DESCRIPTION
**Pull request by Facund0Fernandez**

### CODE REVIEW

File: src/components/Layout/index.js [hunk 1]
```diff
@@ -29,26 +29,28 @@ function Layout() {
...
-            <Route exact path="/projects/form" component={ProjectForm} />
-            <Route exact path="/time-sheets" component={TimeSheets} />
-            <Route exact path="/time-sheets/form" component={TimeSheetForm} />
-            <Route exact path="/tasks" component={Tasks} />
-            <Route exact path="/tasks/form" component={TaskForm} />
-            <Route exact path="/">
-              <Redirect to="/home" />
-            </Route>
-          </Switch>
+          <React.Suspense fallback={<span>Loading</span>}>
```
Comment (GuidoCerioni) [inline src/components/Layout/index.js:32]: have you tried our loadingscreen component here? i think we can use it

**Review by Gearyandres94 — APPROVED** [review]
🚀

**Review by GuidoCerioni — APPROVED** [review]
Nice!

**Review by vadalamati — COMMENTED** [review]
Good job!

**Review by vadalamati — APPROVED** [review]

**Pull request merged**